### PR TITLE
Implement Vanguard IPC connection

### PR DESCRIPTION
**Pull request by mxmar**
Solves: https://app.clickup.com/t/1jtkkb2

Signed-off-by: mxmar <macht4tech@gmail.com>

### CODE REVIEW

**Review by atif-konasl — APPROVED** [review]

File: cmd/orchestrator/main.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ import (
 )
 
 var appFlags = []cli.Flag{
-	cmd.VanguardGRPCEndpoint,
+	cmd.VanguardRPCEndpoint,
```
[the remark this answers is not in the document]
Comment (mxmar) [reply cmd/orchestrator/main.go]: done

File: orchestrator/vanguardchain/client/client.go [hunk 2]
```diff
@@ -31,23 +38,37 @@ type GRPCClient struct {
 	validatorClient ethpb.BeaconNodeValidatorClient
 }
 
-// Dial connects a client to the given URL.
-func Dial(ctx context.Context, rawurl string, grpcRetryDelay time.Duration,
+// Dial connects a client to the given URL or socket path.
+func Dial(ctx context.Context, rpcAddress string, grpcRetryDelay time.Duration,
```
[the remark this answers is not in the document]
Comment (mxmar) [reply orchestrator/vanguardchain/client/client.go]: done

File: cmd/orchestrator/usage.go [hunk 3]
```diff
@@ -58,7 +58,7 @@ var appHelpFlagGroups = []flagGroup{
 			cmd.WSEnabledFlag,
 			cmd.WSListenAddrFlag,
 			cmd.WSPortFlag,
-			cmd.VanguardGRPCEndpoint,
+			cmd.VanguardRPCEndpoint,
```
[the remark this answers is not in the document]
Comment (mxmar) [reply cmd/orchestrator/usage.go]: done

**Comment by mxmar** [comment]
All conversation and issues have been solved. Because of rebase complexity I've decided to move this PR to https://github.com/lukso-network/lukso-orchestrator/pull/95

**Pull request closed without merging**